tests/queries/0_stateless/01420_format_row.sql

```diff
@@ -0,0 +1,4 @@
+select formatRow(CSV, number, 'good') from numbers(3);
```

Comment:
I think that it's better to implement it without any special syntax support, so it will be

`select formatRow('CSV', number, 'good') from numbers(3);`

Reply:
Done